README.md

```diff
@@ -36,6 +36,13 @@ Add Firebase to your project if you haven't already ([Android](https://firebase.
 
 See [Add Firebase Crashlytics to your app](https://firebase.google.com/docs/crashlytics/get-started?platform=android#add-sdk) and follow the instructions to set up your app correctly.
 
+### iOS
+
+See [Add Firebase Crashlytics to your app](https://firebase.google.com/docs/crashlytics/get-started?platform=ios#add-sdk) and follow the instructions to set up your app correctly.
```

Comment:
Unfortunately this is not correct. The `FirebaseCrashlytics` pod is already added and configured by the plugin.